backend/de.metas.business/src/main/java/de/metas/banking/BankAccount.java

```diff
@@ -93,4 +106,12 @@ public boolean isAccountNoMatching(@NonNull final String accountNo)
 				|| postAcctNoCleaned.equals(SEPA_CreditorIdentifier);
 
 	}
+
+	public boolean isAddressComplete()
+	{
+		return Check.isNotBlank(accountStreet) &&
+				Check.isNotBlank(accountZip) &&
+				Check.isNotBlank(accountCity) &&
+				Check.isNotBlank(accountCountry);
```

Comment:
not sure about accountCountry..might be empty if only domestic transactions are anticipated..